shared/src/main/scala/com/raquo/domtypes/generic/defs/tags/SvgTags.scala

```diff
@@ -0,0 +1,733 @@
+package com.raquo.domtypes.generic.defs.tags
+
+import com.raquo.domtypes.generic.builders.TagBuilder
+
+trait SvgTags[T[_ <: DomElement], DomElement, SvgElement <: DomElement,
```

Comment:
TODO for myself: Do we need `DomElement` in the type signature at all? Seems like we could get by with just SvgElement